### ISSUE TITLE
[$50][MSFT-220] [Edge] The NVDA still read item name when user try to mouse hover outside area of them

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2879]
**Steps to Reproduce**
1. Navigate to: https://accounts.topcoder.com/member
2. Try to mouse hover outside area of any item

**Expected Result**
The NVDA should not read item name when user try to mouse hover outside area of them

**Actual Result**
The NVDA still read item name when user try to mouse hover outside area of them

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Edge

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.2.3 Audio Description or Media Alternative (Prerecorded)
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F25

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096246 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096246 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096246 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096769 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096769 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096769 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@sushilshinde @mishacucicea @codeMinter unable to reproduce this. 

Test URL - https://accounts.topcoder.com/member
Env - Windows/Edge
Screen Reader Used - NVDA

**Comment by drasticdpk** [comment]
Hi @r0hit-gupta , here is the reference video ,  Please try to replicate once again. FYI @veshu @lakshmiathreya please suggest what we do if the issue is not replicate on Developer's machine. Thanks! 
https://monosnap.com/file/RkqNWW9l2iEMBasqp6RK8iajze8IMs

**Comment by lakshmiathreya** [comment]
This is working in Chrome/NVDA @nkumar-topcoder - let's take a call - we can put this On Hold since its an Edge issue? fyi @drasticdpk

**Comment by nkumar-topcoder** [comment]
> This is working in Chrome/NVDA @nkumar-topcoder - let's take a call - we can put this On Hold since its an Edge issue? fyi @drasticdpk

@lakshmiathreya, let's not remove any label. But tag on-hold. Because it's already in prod and verified for chrome.

**Comment by lakshmiathreya** [comment]
As per consensus from stakeholders, we are focusing exclusively on Chrome at this time. Putting this Issue on hold since it work on Chrome. fyi - @nkumar-topcoder @sushilshinde @drasticdpk @tosha5252

**Comment by crazyk07** [comment]
Closing Issue.  Creating another issue to cover the Edge Browser instance.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096769<br/><br/>```This is an automated message for crazyk via Topcoder X```
